test/java/seedu/addressbook/commands/DeleteMCommandTest.java

```diff
@@ -0,0 +1,158 @@
+package seedu.addressbook.commands;
+
+import org.junit.Before;
+import org.junit.Test;
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.person.UniquePersonList.PersonNotFoundException;
+import seedu.addressbook.ui.TextUi;
+import seedu.addressbook.util.TestUtil;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class DeleteMCommandTest {
+
+    private AddressBook emptyAddressBook;
+    private AddressBook addressBook;
+
+    private List<ReadOnlyPerson> emptyDisplayList;
+    private List<ReadOnlyPerson> listWithEveryone;
+    private List<ReadOnlyPerson> listWithSurnameDoe;
+
+    @Before
+    public void setUp() throws Exception {
+        Person johnDoe = new Person(new Name("John Doe"), new Phone("61234567", false),
+                new Email("john@doe.com", false), new Address("395C Ben Road", false), Collections.emptySet());
+        Person janeDoe = new Person(new Name("Jane Doe"), new Phone("91234567", false),
+                new Email("jane@doe.com", false), new Address("33G Ohm Road", false), Collections.emptySet());
+        Person samDoe = new Person(new Name("Sam Doe"), new Phone("63345566", false),
+                new Email("sam@doe.com", false), new Address("55G Abc Road", false), Collections.emptySet());
+        Person davidGrant = new Person(new Name("David Grant"), new Phone("61121122", false),
+                new Email("david@grant.com", false), new Address("44H Define Road", false),
+                Collections.emptySet());
+
+        emptyAddressBook = TestUtil.createAddressBook();
+        addressBook = TestUtil.createAddressBook(johnDoe, janeDoe, davidGrant, samDoe);
+
+        emptyDisplayList = TestUtil.createList();
+
+        listWithEveryone = TestUtil.createList(johnDoe, janeDoe, davidGrant, samDoe);
+        listWithSurnameDoe = TestUtil.createList(johnDoe, janeDoe, samDoe);
+    }
+
+    @Test
+    public void execute_emptyAddressBook_returnsPersonNotFoundMessage() {
```

Comment:
The naming convention for your tests doesn't seem to fit the coding standard, i.e. featureUnderTest_testScenario_expectedBehavior()

Reply:
Noted. Thank you! I will update in the level3/4.